src/app/workflow/versions/versions.component.html

```diff
@@ -162,11 +162,11 @@
 
     <ng-container matColumnDef="verified">
       <th scope="col" mat-header-cell *matHeaderCellDef [matTooltip]="verifiedVersionTooltip">
-        Verified Platforms
+        Verified
         <a class="ds-green" [href]="verifiedLink" target="_blank" rel="noopener noreferrer"><mat-icon>info</mat-icon></a>
       </th>
       <td mat-cell *matCellDef="let version">
-        <span matTooltip="{{ getVerifiedSource(version.name) }}">{{ version.id | verifiedPlatforms: verifiedVersionPlatforms }}</span>
+        <mat-icon *ngIf="version?.verified">check</mat-icon>
```

Comment:
Nit: I think the question mark is redundant, but no need to change -- the file is already inconsistent with its usage of it.